pay-api/src/pay_api/services/eft_service.py

```diff
@@ -67,31 +67,6 @@ def create_invoice(self, payment_account: PaymentAccount, line_items: List[Payme
                        **kwargs) -> None:
         """Do nothing here, we create invoice references on the create CFS_INVOICES job."""
 
-    def apply_credit(self,
```

Comment:
Not needed any more. Most of this is done in the jobs now.

Credit management moved to short name service as it makes more sense to live in there now.